include/factory_data.h

```diff
@@ -28,10 +28,12 @@
  * Do not access fields directly. Use API functions only. */
 typedef struct {
```

Comment:
FYI the only "versioning" of the factory data is the length of the struct, which is assumed to be monotonically increasing. Modifying existing fields will break existing setups.